api4/oauth_test.go

```diff
@@ -57,23 +51,15 @@ func TestCreateOAuthApp(t *testing.T) {
 	CheckNoError(t, resp)
 	CheckCreatedStatus(t, resp)
 
-	if rapp.IsTrusted {
-		t.Fatal("trusted should be false - created by non admin")
-	}
+	require.False(t, rapp.IsTrusted, "trusted should be false - created by non admin")
 
 	oapp.Name = ""
 	_, resp = AdminClient.CreateOAuthApp(oapp)
 	CheckBadRequestStatus(t, resp)
 
-	if r, err := Client.DoApiPost("/oauth/apps", "garbage"); err == nil {
-		t.Fatal("should have failed")
-	} else {
-		if r.StatusCode != http.StatusBadRequest {
-			t.Log("actual: " + strconv.Itoa(r.StatusCode))
-			t.Log("expected: " + strconv.Itoa(http.StatusBadRequest))
-			t.Fatal("wrong status code")
-		}
-	}
+	r, err := Client.DoApiPost("/oauth/apps", "garbage")
+	require.NotNil(t, err)
```

Comment:
there is a noerror function: https://godoc.org/github.com/stretchr/testify/require#Assertions.NoError